common/repo/GitRepo.java

```diff
@@ -169,42 +142,25 @@ public void tagHead(String name) {
     return result.build();
   }
 
+  @Override
   public ImmutableList<File> getUncommittedFiles() {
     ImmutableList.Builder<File> files = ImmutableList.builder();
-    try {
-      Status status = jGit.status().call();
-      status
-          .getAdded()
-          .forEach(
-              added ->
-                  files.add(
-                      File.newBuilder().setAction(File.Action.ADD).setFilename(added).build()));
-      status
-          .getModified()
-          .forEach(
-              changed ->
-                  files.add(
-                      File.newBuilder()
-                          .setAction(File.Action.MODIFY)
-                          .setFilename(changed)
-                          .build()));
-      status
-          .getRemoved()
-          .forEach(
-              removed ->
-                  files.add(
-                      File.newBuilder()
-                          .setAction(File.Action.DELETE)
-                          .setFilename(removed)
-                          .build()));
-      status
-          .getUntracked()
-          .forEach(
-              untracked ->
-                  files.add(
-                      File.newBuilder().setAction(File.Action.ADD).setFilename(untracked).build()));
-    } catch (GitAPIException e) {
-      throw new RuntimeException(e);
+    CommandResult commandResult = runCommand("status --short");
+    ImmutableList<String> lines = splitLines(commandResult.stdout);
+    for (String line : lines) {
+      String[] parts;
+      if (!line.trim().startsWith("M")
```

Comment:
Why only in these cases do you need to remove a space, and why do you later split by spaces?
Can you document by giving an example of the original line?